qcodes/instrument_drivers/Andor/DU401.py

```diff
@@ -0,0 +1,424 @@
+        self.head_model = self.atmcd64d.get_head_model()
+        self.firmware_version = self.atmcd64d.get_hardware_version()[4]
+        self.firmware_build = self.atmcd64d.get_hardware_version()[5]
+        self.x_pixels, self.y_pixels = self.atmcd64d.get_detector()
+
+        # add the instrument parameters
+        def accumulation_cycle_time_parser(ans):
+            return float(ans[1])
+        self.add_parameter('accumulation_cycle_time',
+                           get_cmd=self.atmcd64d.get_acquisition_timings,
+                           set_cmd=self.atmcd64d.set_accumulation_cycle_time,
+                           get_parser=accumulation_cycle_time_parser,
+                           unit='s',
+                           label='accumulation cycle time')
+
+        self.add_parameter('acquisition_mode',
+                           set_cmd=self.atmcd64d.set_acqiusition_mode,
+                           val_mapping={
+                               'single scan': 1,
+                               'accumulate': 2
+                           },
+                           label='acquisition mode')
+
+        self.add_parameter('cooler',
+                           get_cmd=self.atmcd64d.is_cooler_on,
+                           set_cmd=self.set_cooler,
+                           val_mapping=create_on_off_val_mapping(on_val=1, off_val=0),
+                           label='cooler')
+
+        def exposure_time_parser(ans):
+            return float(ans[0])
+        self.add_parameter('exposure_time',
+                           get_cmd=self.atmcd64d.get_acquisition_timings,
+                           set_cmd=self.atmcd64d.set_exposure_time,
+                           get_parser=exposure_time_parser,
+                           unit='s',
+                           label='exposure time')
+
+        self.add_parameter('filter_mode',
+                           get_cmd=self.atmcd64d.get_filter_mode,
+                           set_cmd=self.atmcd64d.set_filter_mode,
+                           val_mapping=create_on_off_val_mapping(on_val=2, off_val=0),
+                           label='filter mode')
+
+        self.add_parameter('number_accumulations',
+                           set_cmd=self.atmcd64d.set_number_accumulations,
+                           label='number accumulations')
+
+        self.add_parameter('read_mode',
+                           set_cmd=self.atmcd64d.set_read_mode,
+                           val_mapping={'full vertical binning': 0})
+
+        min_temperature, max_temperature = self.atmcd64d.get_temperature_range()
+        self.add_parameter('set_temperature',
+                           set_cmd=self.atmcd64d.set_temperature,
+                           vals=Ints(min_value=min_temperature,
+                                     max_value=max_temperature),
+                           unit=u"\u00b0"+'C',
+                           label='set temperature')
+
+        self.add_parameter('shutter_mode',
```

Comment:
isn't it possible to read form the instrument the current value of this parameter?